ansible_collections/arista/avd/roles/eos_designs/templates/mlag/router-bgp.j2

```diff
@@ -14,6 +14,12 @@ router_bgp:
       send_community: all
 {%     if switch.mlag_ibgp_origin_incomplete == true %}
       route_map_in: RM-MLAG-PEER-IN
+{%     endif %}
+{%     if underlay_ipv6 is arista.avd.defined(true) %}
```

Comment:
if rfc5549 is a strict requirement, we should set a switch.underlay_ipv6 fact, based on checks for both in the switch facts. This would make it possible to add other conditions later like when we get ipv6 p2p support some day.